### PR TITLE
Update devops-command-center.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Fixed formatting in the opening bullet list.

### CODE REVIEW

File: source/administration/devops-command-center.rst [hunk 1]
```diff
@@ -42,12 +47,14 @@ Getting Started with Incident Management
...
+* **Playbooks** define the process to guide an incident toward resolution. Playbooks are created in advance by defining the tasks, checklists, and other configuration used during an incident.
 * **Tasks** are the discrete actions to be taken when resolving an incident. As the necessary work is completed, the team should mark the tasks as complete. During an incident, tasks may be assigned to a member of the incident channel to communicate ownership.
-* **Stages** are groups of tasks that help focus efforts on the current stage of the incident. While tasks can be completed in any order, stages are worked through sequentially.
 * The **Commander** is a defined role during the incident. This person is responsible for ensuring forward momentum on resolving the incident. By default, the reporter of the incident starts as the commander, but the commander may be changed at any time to another member of the incident channel.
 * **Slash commands** may be assigned to tasks defined in the playbook to help to streamline common actions.
 
+.. note:: 
+  From v1.3.0, stages have been deprecated. Existing playbooks are not affected and will retain their configuration.
```
Comment (lieut-data) [inline source/administration/devops-command-center.rst:56]:
```suggestion
   From v1.3.0, stages have been renamed to checklists, and serve only as a grouping mechanism for tasks. Tasks from any checklist may now be completed at any time.
```

File: source/administration/devops-command-center.rst [hunk 2]
```diff
@@ -56,13 +63,13 @@ To access Mattermost Incident Management:
 Playbooks and Incidents
 -----------------------
 
-Playbooks and incidents are associated with teams in Mattermost. Starting an incident prompts for a name, description, and playbook. The name and description are used as the initial channel name and channel description, but may be modified later. The selected playbook defines the stages and tasks, how incident statuses are communicated, and whether or not the incident channel is public or private. Read more about `public and private channels <https://docs.mattermost.com/help/getting-started/organizing-conversations.html>`_.
+Playbooks and incidents are associated with teams in Mattermost. Starting an incident prompts for a name, description, and playbook. The name and description are used as the initial channel name and channel description, but may be modified later. The selected playbook defines the tasks and checklists, how incident statuses are communicated, and whether or not the incident channel is Public or Private. Read more about `Public and Private channels <https://docs.mattermost.com/help/getting-started/organizing-conversations.html>`_.
```
Comment (lieut-data) [inline source/administration/devops-command-center.rst:66]: The phrase `stages and tasks` made sense before, but I'm less confident about `tasks and checklists`, especially given the reversed order. I'm wondering if we should just say `tasks` /or/ `checklists`? cc @itao

**Review by lieut-data — APPROVED** [review]
A few things, but non-blocking.

**Review by amyblais — APPROVED** [review]

File: source/administration/devops-command-center.rst [hunk 3]
```diff
@@ -56,13 +63,13 @@ To access Mattermost Incident Management:
...
 
 Playbooks are always private by default. Membership in the playbook is required to configure the playbook or start an incident using that playbook. Once an incident is started, any member of the team may be added to the incident channel. Users who are not members of the team are unable to access playbooks or incidents belonging to that team.
 
-While the commander of an incident has a dedicated, named role, all members of an incident channel have unrestricted access to modify the current state of the incident. This permissive access model ensures the incident is never blocked on any one individual. Use private channels and channel membership to restrict access, or configure the playbook as public if everyone on the team is trusted with access to the incident.
+While the commander of an incident has a dedicated, named role, all members of an incident channel have unrestricted access to modify the current state of the incident. This permissive access model ensures the incident is never blocked on any one individual. Use Private channels and channel membership to restrict access, or configure the playbook as public if everyone on the team is trusted with access to the incident.
```
Comment (crspeller) [inline source/administration/devops-command-center.rst:70]: Probably just my ignorance, but curious why would we capitalize `Private` here?

File: source/administration/devops-command-center.rst [hunk 4]
```diff
@@ -148,13 +155,12 @@ Join an incident by joining the incident channel. If the incident channel is pub
 
 If the incident channel is private, an existing member of the incident channel must invite you. System administrators may join private incident channels without permission.
 
-Working with stages and tasks
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Working with checklists and tasks
```
Comment (crspeller) [inline source/administration/devops-command-center.rst:158]: Are we going to call them checklists? Or sections of one checklist? @itao ?

File: source/administration/devops-command-center.rst [hunk 5]
```diff
@@ -181,12 +187,12 @@ Incident status updates ensure that stakeholders remain informed about the progr
...
  * If this is the first status update and the playbook has a defined template, that template will be pre-populated here.
  * If this is a subsequent status update, the message from the last status update will be pre-populated here.
 5. Optionally set a reminder to prompt for the next status update.
- * If this is the first status update and the playbook has a defined default reminder timer, that timer will be pre-selected here.\
- * If this is a subsequent status update, the last reminder timer will be pre-populated here.\
+ * If this is the first status update and the playbook has a defined default reminder timer, that timer will be pre-selected here.
+ * If this is a subsequent status update, the last reminder timer will be pre-populated here.
 6. Select **Update Status** to post your status update.
  * Status updates are posted to the incident channel as a message from the user providing the status update.
  * If the playbook has a defined broadcast channel, status updates are copied to the broadcast channel as a message from the incident bot.
```
Comment (crspeller) [inline source/administration/devops-command-center.rst:198]: Below here:
- Ending an incident needs updating. Reference to stage and it's simpler now. No need to be on last stage, etc. 
- Same with restart. 
- Stages still referenced in the slash commands section. 

Unrelated:
- Why are we documenting the test data command on user facing documentation?

**Review by crspeller — CHANGES_REQUESTED** [review]

File: source/administration/devops-command-center.rst (hunk 2 above)
Comment (justinegeffen) [inline source/administration/devops-command-center.rst:66]:
```suggestion
Playbooks and incidents are associated with teams in Mattermost. Starting an incident prompts for a name, description, and playbook. The name and description are used as the initial channel name and channel description, but may be modified later. The selected playbook defines the process (using tasks or checklists), how incident statuses are communicated, and whether or not the incident channel is Public or Private. Read more about `Public and Private channels <https://docs.mattermost.com/help/getting-started/organizing-conversations.html>`_.
```

File: source/administration/devops-command-center.rst (hunk 4 above)
Comment (justinegeffen) [reply source/administration/devops-command-center.rst:158]: Good question. We can change this to "Working with tasks" - it's also more scalable. Thoughts?

File: source/administration/devops-command-center.rst [hunk 6]
```diff
@@ -148,13 +155,12 @@ Join an incident by joining the incident channel. If the incident channel is pub
 
 If the incident channel is private, an existing member of the incident channel must invite you. System administrators may join private incident channels without permission.
 
-Working with stages and tasks
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Working with checklists and tasks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Any member of the incident channel may work with stages and tasks:
+Any member of the incident channel may work with checklists and tasks:
```
Comment (justinegeffen) [inline source/administration/devops-command-center.rst:161]:
```suggestion
Any member of the incident channel can work with tasks:
```

File: source/administration/devops-command-center.rst (hunk 5 above)
Comment (justinegeffen) [reply source/administration/devops-command-center.rst:198]: Good question re: the test data section. When this was moved across from the Gitbook docs to here I left it in - however, it's very easy to move it back to the repo/Gitbook and have it in the development section. Thoughts?

File: source/administration/devops-command-center.rst (hunk 4 above)
Comment (crspeller) [reply source/administration/devops-command-center.rst:158]: @justinegeffen Sounds good!

File: source/administration/devops-command-center.rst (hunk 5 above)
Comment (crspeller) [reply source/administration/devops-command-center.rst:198]: Moving it back sounds good.

Comment (justinegeffen) [reply source/administration/devops-command-center.rst:198]: It's moved. :)

File: source/administration/devops-command-center.rst [hunk 7]
```diff
@@ -146,15 +153,14 @@ Joining an incident
 
 Join an incident by joining the incident channel. If the incident channel is public, you may join the channel without permission. Search for and join the channel via **Browse Channels**.
 
-If the incident channel is private, an existing member of the incident channel must invite you. System administrators may join private incident channels without permission.
+If the incident channel is private, an existing member of the incident channel must invite you. System Admins may join private incident channels without permission.
 
-Working with stages and tasks
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Working with checklists and tasks
```
Comment (justinegeffen) [inline source/administration/devops-command-center.rst:158]:
```suggestion
Working with tasks
```

**Comment by justinegeffen** [comment]
@crspeller, could you re-review when you have a moment?

**Review by agarciamontoro — APPROVED** [review]
:tada:

**Pull request merged**